spec/files/osms/in/5Zone_2.osm

```diff
@@ -344,7 +344,7 @@ OS:WindowMaterial:Gas,
 OS:WindowMaterial:Glazing,
   {e51b80d0-fd88-4461-83e0-a74c3b42d149}, !- Handle
   CLEAR 3MM,                              !- Name
-  Spectral,                               !- Optical Data Type
+  SpectralAverage,                        !- Optical Data Type
```

Comment:
This needed fixing. I believe the OSM came from the _idf_to_osm_ measure, so maybe something with reverse translation.

Reply:
The reverse translation looks pretty straightforward:
https://github.com/NREL/OpenStudio/blob/develop/src/energyplus/ReverseTranslator/ReverseTranslateWindowMaterialGlazing.cpp#L56

Do you happen to have the IDF file?  I could see if I can reproduce and file a bug?

Reply:
The culprit OSM was indeed generated from _idf_to_osm_, but the translated IDF is not a straightforward derivative of the original [5Zone](https://github.com/NREL/EnergyPlus/blob/b03bdf040164a5ca370b6ea69bf41e08423ad0f6/testfiles/5ZoneAirCooled.idf#L424) lineage. It was preprocessed by a 3rd-party EnergyPlus-based frontend (which I won't name), hence the occurrences of "Spectral" (and not "SpectralAverage"). My bad.